sniffr/breeds/routes.py

```diff
@@ -0,0 +1,15 @@
+from flask import Blueprint, request, jsonify
```

Comment:
Yeah, I noticed that as well. I just thought it might be less redundant and more descriptive if we shortened it. However, if this is what we agree on, I can push the change :)